proto/collection.proto

```diff
@@ -1,69 +1,70 @@
 // collection.proto
 syntax = "proto3";
 
-import "google/protobuf/timestamp.proto";
+package com.gu.mobile.mapi.models;
+
 import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
 
-package com.gu.mobile.mapi.models;
 option swift_prefix="Blueprint";
 
 message Palette {
-    string accent_colour = 1;
-    string background = 2;
-    string comment_count = 3;
-    string element_background = 4;
-    string headline = 5;
-    string immersive_kicker = 6;
-    string main = 7;
-    string media_background = 8;
-    string media_icon = 9;
-    string meta_text = 10;
-    string pill = 11;
-    string pillar = 12;
-    string secondary = 13;
-    string shadow = 14;
-    string top_border = 15;
+  string accent_colour = 1;
+  string background = 2;
+  string comment_count = 3;
+  string element_background = 4;
+  string headline = 5;
+  string immersive_kicker = 6;
+  string main = 7;
+  string media_background = 8;
+  string media_icon = 9;
+  string meta_text = 10;
+  string pill = 11;
+  string pillar = 12;
+  string secondary = 13;
+  string shadow = 14;
+  string top_border = 15;
 }
 
 message Links {
-    string related_uri = 1;
-    string short_url = 2;
-    string uri = 3;
-    string web_uri = 4;
+  string related_uri = 1;
+  string short_url = 2;
+  string uri = 3;
+  string web_uri = 4;
 }
 
 enum MediaType {
-    MEDIA_TYPE_VIDEO = 0;
-    MEDIA_TYPE_AUDIO = 1;
-    MEDIA_TYPE_IMAGE = 2;
+  MEDIA_TYPE_VIDEO_UNSPECIFIED = 0;
```

Comment:
@DavidLawes  just out of curiosity, why it is renamed?

Reply:
Ah, no worries. I see your comment below

Reply:
I think this is a valid change to make. If a zero value enum is defined (e.g. in the MAPI) response then I think it's automatically excluded from the data that's actually returned to the client in the response payload. Naming zero value enums as UNSPECIFIED is also defined in the style guide: https://protobuf.dev/programming-guides/style/#enums